tests/ragger/app/client.py

```diff
@@ -156,3 +194,57 @@ def eip712_filtering_show_field(self, name: str, sig: bytes):
         with self._send(self._cmd_builder.eip712_filtering_show_field(name, sig)):
             pass
         assert self._recv().status == 0x9000
+
+    def send_fund(self,
+                  bip32_path: list[Optional[int]],
+                  nonce: int,
+                  gas_price: int,
+                  gas_limit: int,
+                  to: bytes,
+                  amount: float,
+                  chain_id: int,
+                  screenshot_collection: str = None):
+        for chunk in self._cmd_builder.send_fund(bip32_path,
+                                                 nonce,
+                                                 gas_price,
+                                                 gas_limit,
+                                                 to,
+                                                 amount,
+                                                 chain_id):
+            with self._send(chunk):
+                nav_ins = NavIns(NavInsID.RIGHT_CLICK)
+                final_ins = [ NavIns(NavInsID.BOTH_CLICK) ]
+                target_text = "and send"
+                if screenshot_collection:
+                    self._nav.navigate_until_text_and_compare(nav_ins,
+                                                              final_ins,
+                                                              target_text,
+                                                              ROOT_SCREENSHOT_PATH,
+                                                              screenshot_collection)
+                else:
+                    self._nav.navigate_until_text(nav_ins,
+                                                  final_ins,
+                                                  target_text)
+            assert self._recv().status == 0x9000
+
+    def get_challenge(self) -> int:
+        with self._send(self._cmd_builder.get_challenge()):
+            pass
+        resp = self._recv()
+        return self._resp_parser.challenge(resp.data)
```

Comment:
We never want to test the `get_challenge` APDU returning an error ? Like a bad state / incorrect flow / etc ?
In wich case OK for having the implicit `assert 0x9000` + `unpack`

Reply:
This APDU cannot return an error.